ChangeLog.md

```diff
@@ -89,6 +89,10 @@
 
   * Add example course question templates (James Balamuta).
 
+  * Add extra `/course*` mount points (Tim Yang).
```

Comment:
Not sure how this got here. Thought this was in my old PR.

Reply:
Probably when you merged master, this is *ok* I think?

Reply:
Yeah, those lines will come in from master mergers.

Note that we are actually not worrying about updating ChangeLog anymore. We now use the git log as the source of truth. At some point soon I would like to add a bot to automatically mirror the git log into the ChangeLog file, but for now we are just neglecting ChangeLog.